reana_client/api/client.py

```diff
@@ -629,30 +629,23 @@ def diff_workflows(workflow_id_a, workflow_id_b,
         raise e
 
 
-def open_interactive_session(workflow, access_token, image=None, port=None):
+def open_interactive_session(workflow, access_token,
+                             interactive_session_type):
     """Open an interactive session inside the workflow workspace.
 
     :param workflow: Workflow which workspace will be available inside the
         interactive session.
     :param access_token: Workflow owner REANA access token.
-    :param image: Image to use in the interactive session, by default it is
-        ``jupyter/scipy-notebook``.
-    :param port: Port exposed by the service run inside the container spawned
-        with ``image``, by default ``8888`` for Jupyter notebooks.
+    :param interactive_session_type: Type of interactive session to spawn.
```

Comment:
Going for type `jupyter` is good, but it'll bd still interesting to offer an option to users to provide a customised image that will match the `jupyter` type protocol... E.g. there are [lots of stacks](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html) that people may want to choose from, including a wish to specify their own version. Kind of like:

```console
$ reana-client open jupyter     # detault, e.g. jupyter/scipy-notebook:latest
$ reana-client open jupyter --image jupyter/tensorflow-notebook:2343e33dec46
```